pacman-nodejs-app/docs/pacman-nodejs-app-aws-migration.md

```diff
@@ -0,0 +1,532 @@
+# Pac-Man AWS Migration
+
+This tutorial walks you through the steps to perform a Pac-Man migration from an AWS cluster A to a GKE cluster B using GKE for hosting. The tutorial documents the steps
```

Comment:
I'm not sure what to say to specify that GKE cloud DNS needs to know all of the associated NS records/ip addresses. That is what I meant to specify.